app/helpers/application_helper.rb

```diff
@@ -117,6 +117,10 @@ def website_nav?
     policy(Website).show?
   end
 
+  def sponsor_nav?
```

Comment:
I am guessing that this is cruft now and can be removed?

Reply:
Yep. I cleaned it out.